Mathlib/Analysis/Calculus/Gradient/Basic.lean

```diff
@@ -24,16 +24,16 @@ is restricted to `s`. We also have
 
 ## Main results
 
-This file contains the following parts of gradient.
-* the definition of gradient.
+This file develops the following aspects of the theory of gradients:
+* definitions of gradients, both within a set and on the whole space.
```

Comment:
I kind of like your changes, but I don't like how all bullet points start with "theorems establishing", "the theorems about"... can you remove those parts, i.e. make it "the gradient of constant functions"?

Reply:
Sorry for the delay in addressing this. I've done so now. Originally, I was trying to not stray too far from pure typo-fixes, but I agree that your suggestion is for the better, so why not.